src/Blog/Handler/ListPostsHandler.php

```diff
@@ -108,7 +107,7 @@ private function prepareView(string $tag, array $entries, stdClass $pagination):
         return [
             ...$view,
             ...[
-                'posts'      => $entries,
+                'posts'      => ArrayUtils::iteratorToArray($entries, false),
```

Comment:
Possible future improvement: ensure that `$recursive = true` is considered when inferring a return type.

It did bite me :D